Dockerfile

```diff
@@ -68,6 +68,10 @@ COPY        --from=download /usr/bin/ff* /usr/bin/
 
 # Build devevelopment image
 FROM        base as dev
+RUN         apt-get install -y --no-install-recommends --allow-unauthenticated \
```

Comment:
Was this just missed before or needed as part of this upgrade?

Reply:
This is needed now because I couldn't update Rails without installing nokogiri, which requires build dependencies.